nimbus/examples/experiment.rs

```diff
@@ -226,7 +227,8 @@ fn main() -> Result<()> {
         ("update-experiments", _) => {
             println!("======================================");
             println!("Updating experiments");
-            nimbus_client.update_experiments()?;
+            nimbus_client.fetch_experiments()?;
+            nimbus_client.apply_pending_experiments()?;
```

Comment:
I think it would be idea if we split this into 2 commands (ie, `fetch-experiments` and `apply-...`) just so this tool ends up being able to reproduce the state real apps find themselves in). I'm not too bothered if you want to do this later though because for this to make sense we probably want to print the state of the DB (ie, so the user could see there are fetched-but-not-applied experiments pending) and I'm not going to block you on that kind of change (but would welcome it ;) At a minimum though, I think changing the `println` above is worthwhile to use the same terminology (eg, `Fetching and applying experiments` or similar) just to set expectations correctly.

Reply:
I've added the two additional commands; however, I'm going to file an issue to consider removing the [initial update][1] and adding a subcommand for `set_local_experiments` (to set a file).

[1]: https://github.com/mozilla/nimbus-sdk/pull/78/files#diff-ab0d747cc0a25ee15aaa963c028e831d3bac3cff22997e0b4784ca4993cfc5d6R202-R203

Reply:
Filed [SDK-162][1] for adding commands for testing startup.

[1]: https://jira.mozilla.com/browse/SDK-162